example/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  // "extends": "@tsconfig/node16/tsconfig.json",
+  // "extends": "@tsconfig/node-lts/tsconfig.json",
```

Comment:
Why is this commented?

Reply:
Dunno, found it like this - I'll clean it up since it doesn't seem to be of use